azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -510,6 +518,7 @@ def __reset_agent_cpu_quota():
             logger.info("Resetting agent's CPUQuota")
             if CGroupConfigurator._Impl.__try_set_cpu_quota(''):  # setting an empty value resets to the default (infinity)
                 CGroupsTelemetry.set_track_throttled_time(False)
+                _log_cgroup_info('CPUQuota: {0}', systemd.get_unit_property(systemd.get_agent_unit_name(), "CPUQuotaPerSecUSec"))
```

Comment:
aren't we logging this at line 426?

Reply:
yes, that will be logged when cgroup initializes but not after resetting the quota.

Reply:
ok